packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/errors.py

```diff
@@ -11,3 +11,7 @@ class CatalogItemNotFoundError(CatalogApiBaseError):
 
 class CatalogForbiddenError(CatalogApiBaseError):
     msg_template = "Insufficient access rights for {name}"
+
+
+class CatalogServiceError(CatalogApiBaseError):
```

Comment:
this name is a bit confusing. at first I was not sure if this was a the catalog service or some services that it returns. Also by the fact that this is at the bottom of the hierarchy. 
So questions:
- should this not be the base of all the other errors?
- why is it descending from CatalogApiBaseError?